src/components/DataSession.vue

```diff
@@ -28,9 +28,20 @@ async function addOperation(operationDefinition) {
 	await fetchApiCall({url: url, method: 'POST', body: operationDefinition, successCallback: emit('reloadSession'), failCallback: handleError})
 }
 
+const displayImages = (data) => {
+	images.value = data.input_data
+	console.log('images.value', images.value)
```

Comment:
Extraneous console log? I think having this inline like it was before is just fine if it's a one-liner

Reply:
oopsi